coil-base/src/main/java/coil/disk/DiskCache.kt

```diff
@@ -59,12 +65,13 @@ interface DiskCache {
      * disk cache. To modify the contents of those files, use [edit].
      */
     interface Snapshot : Closeable {
+        val fileSystem: FileSystem
```

Comment:
I'm not certain, but I think the `FileSystem` should be implied by `DiskCache.fileSystem` otherwise there could be confusion (i.e. can each individual `Snapshot`/`Editor` have a different `FileSystem` from `DiskCache.fileSystem`?). What do you think?